<launch>
    <node pkg="sigma7" type="multiSigmaDevices" name="multiSigmaDevices" output="screen">

        <param name="frequency" value="500" />

        <!--sigma can simulate a button with the gripper. THat is when yoy
        close the gripper it resists a bit at the end and and springs back
        when you release it.-->
        <param name="left_hand" value="/sigma1" />
        <param name="right_hand" value="/sigma0" />
        <param name="id_left" value="1" />
        <param name="id_right" value="0" />

        <param name="enable_gripper_button" value="true" />

        <!--we can lock the orientation when the pedal is released. This is
        useful for teleoperation-->
        <param name="lock_orientation" value="false" />

        <!-- The topic from which the node reads wrenches to be applied by
        the device-->
        <param name="wrench_topic" value="/sigma/force_feedback"/>

    </node>
</launch>
